internal/services/oracle/oracle_exascale_database_virtual_machine_cluster_data_source.go

```diff
@@ -0,0 +1,571 @@
+	SingleClientAccessNameDnsRecordId        string                                              `tfschema:"single_client_access_name_dns_record_id"`
+	SingleClientAccessNameIpIds              []string                                            `tfschema:"single_client_access_name_ip_ids"`
+	SingleClientAccessNameListenerPortTcp    int64                                               `tfschema:"single_client_access_name_listener_port_tcp"`
+	SingleClientAccessNameListenerPortTcpSsl int64                                               `tfschema:"single_client_access_name_listener_port_tcp_ssl"`
+	Shape                                    string                                              `tfschema:"shape"`
+	SnapshotFileSystemStorage                []ExascaleDatabaseVirtualMachineClusterStorageModel `tfschema:"snapshot_file_system_storage"`
+	SshPublicKeys                            []string                                            `tfschema:"ssh_public_keys"`
+	SubnetId                                 string                                              `tfschema:"subnet_id"`
+	SubnetOcid                               string                                              `tfschema:"subnet_ocid"`
+	SystemVersion                            string                                              `tfschema:"system_version"`
+	TimeZone                                 string                                              `tfschema:"time_zone"`
+	TotalEcpuCount                           int64                                               `tfschema:"total_ecpu_count"`
+	TotalFileSystemStorage                   []ExascaleDatabaseVirtualMachineClusterStorageModel `tfschema:"total_file_system_storage"`
+	VirtualIpIds                             []string                                            `tfschema:"virtual_ip_ids"`
+	VirtualMachineFileSystemStorage          []ExascaleDatabaseVirtualMachineClusterStorageModel `tfschema:"virtual_machine_file_system_storage"`
+	VnetId                                   string                                              `tfschema:"virtual_network_id"`
+	ZoneOcid                                 string                                              `tfschema:"zone_ocid"`
+}
+
+type IormConfigModel struct {
+	DatabasePlans    []ExascaleDatabaseIormConfigModel `tfschema:"database_plans"`
+	LifecycleDetails string                            `tfschema:"lifecycle_details"`
+	LifecycleState   string                            `tfschema:"lifecycle_state"`
+	Objective        string                            `tfschema:"objective"`
+}
+
+type ExascaleDatabaseIormConfigModel struct {
+	DatabaseName    string `tfschema:"database_name"`
+	FlashCacheLimit string `tfschema:"flash_cache_limit"`
+	Share           int64  `tfschema:"share"`
+}
+
+type ExascaleDatabaseVirtualMachineClusterStorageModel struct {
+	TotalSizeInGb int64 `tfschema:"total_size_in_gb"`
+}
```

Comment:
This can be moved to `oracle_exascale_database_virtual_machine_cluster.go` since it is used by both the resource and the datasource.